api/adobe_vendor_id.py

```diff
@@ -185,6 +185,7 @@ def handle_signin_request(self, data, standard_lookup, authdata_lookup):
         try:
             data = parser.process(data)
         except Exception, e:
+            logging.error("Error processing %s", data, exc_info=e)
```

Comment:
Without this, we know when there are problems processing Adobe Vendor ID SignIn requests but we don't know the payloads that caused the problems.